manifests/config.pp

```diff
@@ -102,7 +102,7 @@
   # Just convert to yaml
   file {"${config_path}/${config_file}":
     ensure  => $ensure,
-    content => inline_template('<%= @cobbler_config.to_yaml %>'),
```

Comment:
Also when generating Cobbler YAML file the default @cobbler_config.to_yaml the integer values are passed as string to Cobbler. Which is causing configuration problems due cobbler missing integer conversions from string.
It was second reason why yaml was made as function.